tests/conftest.py

```diff
@@ -138,6 +138,16 @@ def random_graph_edgelist_fixture():
     return edgelist
 
 
+@pytest.fixture(name="full_random_graph_edgelist", scope="module")
+def full_random_graph_edgelist_fixture():
```

Comment:
This looks very similar to `random_graph_edgelist_fixture`. Why do we need this?

Reply:
random_graph_edgelist_fixture randomizes the edges but not the markers (the first half are marker A and second half are marker B). full_random_graph_edgelist also randomizes markers from A to G and we need multiple markers to be able to check the effect of removing low-count markers on colocalization scores.

Reply:
Aha. You are right. Thanks.